src/init.js

```diff
@@ -1,27 +1,53 @@
-const { promises: fs } = require('fs');
+var fs = require('fs');
+const { promises: pfs } = require('fs');
+var https = require('https');
```

Comment:
Can we use const (or let, if needed)?

Reply:
Yes.  I apologize, I'm started programming 40 years ago, and this is my first time coding in Java, so I'm a little unfamiliar with the language and conversions.  I don't see why const can't be used everywhere here.  Changed.